src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -196,6 +196,16 @@ public function getConfigTreeBuilder(): TreeBuilder
                                 ->end()
                                 ->booleanNode('entity_manager_handler')
                                     ->defaultNull()
+                                    ->setDeprecated(
+                                        'k911/swoole-bundle',
+                                        '0.8.4',
```

Comment:
I can change that after merge, however it'll be at least `0.9.0` version. So just a note for me.